src/ORM/Query.php

```diff
@@ -1275,4 +1275,22 @@ protected function _decorateResults($result)
 
         return $result;
     }
+
+    /**
+     * Selects all fields in the schema with the exception of the fields you pass
+     * in the $excludedFields parameter.
+     *
+     * @param array $excludedFields fields you don't want to select
+     * @param bool $overwrite whether to reset fields with passed list or not
+     * @return $this
+     */
+    public function selectAllExcept(array $excludedFields = [], $overwrite = false)
```

Comment:
This should receive an instance of `Table` `Association` or `TableSchema` as first argument